### ISSUE TITLE
Offer to help

### ISSUE THREAD

**Issue by TheBlindEye** [ThePatr10t/yall-Qaeda#2]
Hi I started a fork because I was working on a similar project and I prefer your split screen layout much better than linking from the map like I am. I don't see any reason to duplicate work however, so I would love to lend a hand with your project if you're interested. 

here is my map: https://www.google.com/maps/d/u/0/edit?mid=1nLBMu21IZTQ82ls6NdG-0pjIA9kXxmAk&usp=sharing

I can generate the kml for you for the files I am using on my map. I have permission from www.tommycarstensen.com to deeplink as I am (as long as we give some credit) and we should be able to embed them easily with `<video>` tags rather than linking out.

Basically, let me know if you want to team up and combine efforts.

**Comment by ThePatr10t** [comment]
Totally down to collaborate! I love what you built. Super slick. Mines hella simple but I can't take credit for it. Its just based around a google reference implementation.

Totally down for doing video embeds, so long as there's bandwidth available. Id have to get that working with this wonky maps iframe interface, but Im sure its possible.

Take a look at the file "uploaded", and feel free to contribute additions to it.

Also, this simple JS code reads the results from the KML and acts depending on the data source. It might be that we can add a line for <video> tags, given urls that don't match some list of tube sites.

https://github.com/ThePatr10t/yall-Qaeda/blob/6a2199b16fc75a7922a6f9681ea6952ff580593e/index.html#L89

I alllmost got dailymotion working. No dice

**Comment by TheBlindEye** [comment]
OK perfect I think the best way to move forward is still with a fork and then pull requests, but I am not terribly familiar with git or github. Does that sound right to you? 

When I update the "uploaded" file would you like to give youtube links priority? I think that makes the most sense, to avoid running into bandwidth issues.

**Comment by osint45** [comment]
I'd also be interested in contributing. I've been planning a timeline tool for these videos and would love to look into #3 which sounds like what I was picturing.

**Comment by ghost** [comment]
Echoing other replies here, I would love to assist assuming I could add value. Im a dB/ front-end BI developer who works in tools like power bi, tableau etc. Last few years I've been learning html and python. Specifically the dash package which has a lot of helpful front end ui controls and may be useful. If not, no biggie, just consider this a vote for issue #3 in getting a timeline control in there if possible :)

**Comment by Xesme** [comment]
Hi, I am a front end / full-stack developer in web and mobile applications. I am also an experienced UX/UI designer. I noticed your site doesn’t work on mobile. Can I help you out with design and mobile implementations?

**Comment by devyboy** [comment]
Echoing the others, I also dabble in front-end (mainly React/React Native), a little bit of fullstack, and some UI/UX here and there. Would love to assist with this project in any way.
